src/main/java/ui/input/InputView.java

```diff
@@ -0,0 +1,40 @@
+package ui.input;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+
+    private static final String INPUT_PLAYERS_NAME = "게임에 참여할 사람의 이름을 입력하세요.(쉼표 기준으로 분리)";
+    private static final String RECEIVE_MORE_CARD = "%s는 한장의 카드를 더 받겠습니까?(예는 y, 아니오는 n)";
+    private static final String RECEIVE_CARD_COMMAND = "y";
+    private static final String NOT_RECEIVE_CARD_COMMAND = "n";
+
+    private static final Scanner SCANNER = new Scanner(System.in);
+
+
+    public static List<String> getPlayersName() {
+        System.out.println(INPUT_PLAYERS_NAME);
+        return treatSpace(SCANNER.nextLine());
+    }
+
+    private static List<String> treatSpace(String input) {
+        List<String> names = Arrays.asList(input.split(","));
+        for (int i = 0, size = names.size(); i < size; i++) {
+            names.set(i, names.get(i).trim());
+        }
```

Comment:
```suggestion
        return Arrays.stream(input.split(","))
            .map(String::trim)
            .collect(Collectors.toList());
```

요렇게는 어떤가요 ? 다른 곳에서 stream 잘 써주셔서 한번 추천드려요 😉